go/libraries/doltcore/sqle/logictest/DoltRegressionsJenkinsfile

```diff
@@ -0,0 +1,91 @@
+                            sh "rm -rf temp && mkdir temp"
+                            script {
+                                def j = env.TEST_N_TIMES.toInteger()
+                                for (int i = 1; i <= j; ++i) {
+                                    sh "go run . run ../../../../../../sqllogictest/test/select1.test > temp/results${i}.log"
+                                    sh "go run . parse \"$DOLT_VERSION\" temp/results${i}.log > temp/parsed${i}.json"
+                                }
+                            }
+                        }
+                        dir ("dolt-sql-performance") {
+                            sh "dolt checkout nightly"
+                            script {
+                                def j = env.TEST_N_TIMES.toInteger()
+                                for (int i = 1; i <= j; ++i) {
+                                    sh "dolt table import -u nightly_dolt_results ../go/libraries/doltcore/sqle/logictest/main/temp/parsed${i}.json"
+                                    sh "dolt add nightly_dolt_results"
+                                    sh "dolt commit -m \"update dolt sql performance results\""
+                                }
+                            }
+                            sh "rm -f mean.csv"
+                            sh "dolt sql -r csv -q \'select version, test_file, line_num, avg(duration) as mean_duration, result from dolt_history_nightly_dolt_results where version=\"${DOLT_VERSION}\" group by line_num;\' > mean.csv"
+                            sh "dolt table import -u nightly_dolt_mean_results mean.csv"
+                            sh "dolt add nightly_dolt_mean_results"
+                            sh "dolt commit -m \"update dolt sql performance mean results\""
+                            sh "dolt push origin nightly"
+                            sh "dolt checkout regressions"
+                            sh "dolt merge nightly"
+                            sh "dolt add ."
+                            sh "dolt commit -m \"merge nightly\""
+                            sh "dolt push origin regressions"
+                            script {
+                                env.DURATION_REGRESSIONS = sh(returnStdout: true, script: "dolt sql -q 'select * from releases_nightly_duration_change' -r csv | wc -l | tr -d '[:space:]'")
+                                env.RESULT_REGRESSIONS = sh(returnStdout: true, script: "dolt sql -q 'select * from releases_nightly_result_change' -r csv | wc -l | tr -d '[:space:]'")
+                            }
+                            sh "if [ $DURATION_REGRESSIONS != 1 ]; then echo 'PLEASE FAIL ME!!'; fi"
+                            sh "if [ $RESULT_REGRESSIONS != 1 ]; then echo 'PLEASE FAIL ME!!'; fi"
+                            sh "if [ $DURATION_REGRESSIONS = 1 ]; then echo 'PLEASE PASS ME!!'; fi"
+                            sh "if [ $RESULT_REGRESSIONS = 1 ]; then echo 'PLEASE PASS ME!!'; fi"
```

Comment:
I will update these before merging this as well... We want to send an email if the regression tests fail